File: app/src/main/java/com/cray/software/passwords/interfaces/Module.java
package com.cray.software.passwords.interfaces;

import android.os.Build;

import com.cray.software.passwords.BuildConfig;

public class Module {
    public static boolean isPro(){
        return BuildConfig.IS_PRO;
    }

    public static boolean isLollipop(){
        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP;
    }

    public static boolean isMarshmallow() {
        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.M;
    }
}

File: app/src/main/java/com/cray/software/passwords/interfaces/Module.kt
package com.cray.software.passwords.interfaces

import android.os.Build

import com.cray.software.passwords.BuildConfig

object Module {
    val isPro: Boolean
        get() = BuildConfig.IS_PRO

    val isLollipop: Boolean
        get() = Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP

    val isMarshmallow: Boolean
        get() = Build.VERSION.SDK_INT >= Build.VERSION_CODES.M
}
